lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -242,26 +245,29 @@ defmodule ExUnit.DocTest do
   end
 
   defp test_name(%{fun_arity: :moduledoc}, m, n) do
-    "module #{inspect m} (#{n})"
+    "module #{inspect(m)} (#{n})"
   end
 
   defp test_name(%{fun_arity: {f, a}}, m, n) do
-    "#{inspect m}.#{f}/#{a} (#{n})"
+    "#{inspect(m)}.#{f}/#{a} (#{n})"
   end
 
   defp test_content(%{exprs: exprs, line: line}, module, do_import) do
-    file     = module.__info__(:compile)[:source] |> Path.relative_to_cwd
+    file = module.__info__(:compile)[:source] |> Path.relative_to_cwd()
     location = [line: line, file: file]
-    stack    = Macro.escape [{module, :__MODULE__, 0, location}]
+    stack = Macro.escape([{module, :__MODULE__, 0, location}])
 
     if multiple_exceptions?(exprs) do
-      raise Error, line: line, module: module,
-                   message: "multiple exceptions in one doctest case are not supported"
+      raise Error,
+        line: line,
+        module: module,
+        message: "multiple exceptions in one doctest case are not supported"
     end
 
-    tests = Enum.map exprs, fn {expr, expected} ->
-      test_case_content(expr, expected, location, stack)
-    end
+    tests =
+      Enum.map(exprs, fn {expr, expected} ->
```

Comment:
it seems we're using parens everywhere... shouldn't it rather be `fn({expr, expected})` then?

Reply:
because we don't put parens in clauses. For example, we don't write:

```
case foo do
  ({:ok, _}) -> ...
end
```